code/game/machinery/dance_machine.dm

```diff
@@ -464,6 +464,7 @@
 				if(!L || !L.client)
 					continue
 				L.stop_sound_channel(CHANNEL_JUKEBOX)
+		use_power(active_power_usage)
```

Comment:
isn't this what the automatic usage stuff is for? or

Reply:
true, i can add the procs to the right calls